src/ray/object_manager/plasma/eviction_policy.h

```diff
@@ -96,8 +97,8 @@ class EvictionPolicy {
   ///
   /// \param store_info Information about the Plasma store that is exposed
   ///        to the eviction policy.
-  /// \param max_size Max size in bytes total of objects to store.
-  explicit EvictionPolicy(PlasmaStoreInfo *store_info, int64_t max_size);
+  /// \param allocator Memory allocator.
+  explicit EvictionPolicy(PlasmaStoreInfo *store_info, IAllocator &allocator);
```

Comment:
This can be const (along with the private field)? 
```suggestion
  explicit EvictionPolicy(PlasmaStoreInfo *store_info, const IAllocator &allocator);
```

Reply:
Seems to only use the const method